.github/.markdownlint.cjs

```diff
@@ -0,0 +1,8 @@
+
+var options = require('../.markdownlint-cli2.cjs');
```

Comment:
```suggestion
const options = require('../.markdownlint-cli2.cjs');
```
Let's not become barbarians just because there is no linting on this file 😉. Now that this is a JS file, can you style this and the other one similarly to how `.eslintrc.js` is syled? That is, use `MD041: false` instead of `"MD041": false` and use single quotes for strings.

Can this file also become a `.markdownlint-cli2.cjs` file instead of `.markdownlint.cjs`? Just for consistency with the other one.